src/com/videojs/providers/RTMPVideoProvider.as

```diff
@@ -463,6 +464,7 @@ package com.videojs.providers{
             switch(e.info.code){
                 case "NetConnection.Connect.Success":
                     _model.broadcastEventExternally(ExternalEventName.ON_RTMP_CONNECT_SUCCESS);
+                    _nc.call("FCSubscribe", null, _src.streamURL); // try to subscribe
```

Comment:
I've tested it with (node-rtsp-rtmp-server)[https://github.com/iizukanao/node-rtsp-rtmp-server], as it does not support FCSubscribe. VOD videos play fine: as the streaming server reports "unknown command" the swf code ignores it.